src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = element.querySelector('.js-Slideshow-slides'),
+    slides = element.querySelectorAll('.js-Slideshow-slide'),
+    nextTrigger = element.querySelector('.js-Slideshow-next'),
+    prevTrigger = element.querySelector('.js-Slideshow-prev'),
+    currentCountElement = element.querySelector('.js-Slideshow-current'),
+    totalCountElement = element.querySelector('.js-Slideshow-total'),
```

Comment:
@mrgerardorodriguez @saralohr 

In general, I'd say it's a good idea to avoid assuming the existence of elements, but I'm also aware that we might be guilty of this pattern in other modules.

Also, it does appear as though there are some excessive calls to `element.querySelectorAll()` in each case where we subsequently wrap the elements with Umbrella's `u()` function.

If we're going to use Umbrella in a module, then we probably don't need `element.querySelectorAll()`.